internal/log/log.go

```diff
@@ -81,41 +81,83 @@ func SetLogLevel(lvl Level) {
 }
 
 // Output format should be: "timestamp•LOG_LEVEL•filename.go•linenumber•output"
-func Fatal(format string, args ...interface{}) {
-	logger.Fatal(format, args...)
+func Fatal(msg interface{}, args ...interface{}) {
+	var msgString string
+	switch m := msg.(type) {
+	case string:
+		msgString = m
+	case error:
```

Comment:
I have not implemented this package yet, but are you finding an use case for passing in error types as format strings? If it's not a common use case, I would prefer that we keep with the string type over the interface type so we get compile time errors rather than runtime errors. If this is an use case, then we should add a `default` statement for the case when the type is not `string` or `error`. I'm not quite sure how to handle it though. I don't think it's a good idea to panic here, but returning an error will also require all logging to then capture errors.

Reply:
The standard log package accepts an error or a string and both are currently being used.  I made this change to allow the package to replace existing logging without having to update calls to use an "log.Error(err.Error())" instead of simply"log.Error(err)".